config/environment.js

```diff
@@ -64,7 +64,11 @@ module.exports = function(environment) {
       hostWhitelist: [/.+/]
     },
 
-    torii: {}
+    torii: {},
+
+    appGenerator: {
+      webHost: process.env.WEB_APP_GENERATOR_HOST || (environment === 'production' ? 'https://open-event-wsgen.herokuapp.com' : 'https://open-event-wsgen-dev.herokuapp.com')
```

Comment:
No need for a nested field, just write it as `webAppGeneratorUrl`

Reply:
I made it nested because we have another button for the android app generator as well. Let me know if I need to make it flat.

Reply:
That will be another field